Allan-breakout-room/Neven/style.css

```diff
@@ -0,0 +1,117 @@
+*{
+    margin:0;
+    padding: 0;
+  }
+
+  .mainheader{
+    width:100%;
+    height: 60px;
+    background:#F0FFFF;
+  }
+  header .width{
+    width:70%;
+    margin:0 auto;
+  }
+  .logo{
+    width: 50px;
+    height: 50px;
+    margin-top:5px;
+  }
+  .navbar{
+    float:right;
+   margin-top:20px;
+  }
+  .btn{
+    text-decoration: none;
+    border:2px solid skyblue;
+    border-radius:10px;
+    font-size:20px;
+    padding:8px;
+  
+    color:skyblue;
+  }
+  .blue{
+    background:skyblue;
+    color: white;
+  }
+  .profile img{
+  margin-top:-80px;
+  margin-left:40px;
+  width: 150px;
+  height:150px;
+
+  }
+  .profile-info{
+    margin-left:225px;
+    margin-top:-60px;
+    margin-bottom:50px;
+  }
+  .follow{
+    position: relative;
+    top: 0;
+    left: 0;
+  }
+  .image1{
+     position: relative;
+    left: 0;
+  }
+  .image2{
+    position: relative;
+    right: 30px;
+  }
```

Comment:
It is not a good practice to use `position: relative` and then use `top` or `left`  ...etc on the same element.
the better  practice is to use `position: absolute` with `left` or `right` property and you should git the parent (in your case it's div) a `position: relative" .

![image](https://user-images.githubusercontent.com/55356198/80698622-6c746c80-8ae3-11ea-8e68-c702db2df46c.png)


that is just my opinion